src/SourceCode.Clay.OpenApi/Builders/ResponseBuilder.cs

```diff
@@ -48,6 +48,9 @@ public class ResponseBuilder
         /// </summary>
         public ResponseBuilder()
         {
+            Headers = new Dictionary<string, Referable<ParameterBody>>();
```

Comment:
StringComparer.Ordinal

Reply:
I will have to write a comparer based on the spec. This will be updated as part of #80.